Index: src/spigot/java/net/cpas/mc/listeners/onLogin.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/spigot/java/net/cpas/mc/listeners/onLogin.java	(date 1575935690000)
+++ src/spigot/java/net/cpas/mc/listeners/onLogin.java	(date 1575936005571)
@@ -68,7 +68,7 @@
 
             /*
             This goes through all groups CPAS gives us, and checks with the group given to see if they are a match
-            if they are, it returns true
+            if they are, it returns true.
              */
 
             for(CpasGroupModel cpasGroupModel : groups) {
